front/app/utils/cl-react-query/fetcher.ts

```diff
@@ -15,30 +15,35 @@ interface Get {
   action: 'get';
   queryParams?: Record<string, any>;
   body?: never;
+  disableCacheOptimization?: boolean;
```

Comment:
Would be nice to add a TSDoc here. to make it clear what this does.
Also less important and more subjective: I often feel it's less confusing if an option is in the positive form, and defaults to `true` , instead of setting something to `true` to disable it. 


```suggestion
  /** When set to true, if the API response returns an array of items, these items will individually be added to the cache in addition to the whole request. Defaults to true. */
  cacheIndividualItems?: boolean;
```

Reply:
Good point! Added the comment.